packages/graphql/src/mutations/marketplace/deployMarketplace.js

```diff
@@ -1,9 +1,11 @@
-import Marketplace from '@origin/contracts/build/contracts/V00_Marketplace'
+import MarketplaceV0 from '@origin/contracts/build/contracts/V00_Marketplace'
+import MarketplaceV1 from '@origin/contracts/build/contracts/V01_Marketplace'
 import txHelper, { checkMetaMask } from '../_txHelper'
 import contracts, { setMarketplace } from '../../contracts'
-const data = Marketplace.bytecode
 
-async function deployMarketplace(_, { token, from, autoWhitelist }) {
+async function deployMarketplace(_, { token, from, autoWhitelist, version }) {
+  const Marketplace = version === '001' ? MarketplaceV1 : MarketplaceV0
```

Comment:
nit: maybe throw an exception if version is unknown so that in future we don't deploy MarketplaceV0 again by mistake :)